test/event/handlers/DiscordMessageLinkHandlerTest.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from "chai";
+import { Constants } from "discord.js";
+import DiscordMessageLinkHandler from "../../../src/event/handlers/DiscordMessageLinkHandler";
+import { SinonSandbox, createSandbox } from "sinon";
+import EventHandler from "../../../src/abstracts/EventHandler";
+import MockDiscord from "../../MockDiscord";
+import MessagePreviewService from "../../../src/services/MessagePreviewService";
+
+describe("DiscordMessageLinkHandler", () => {
+	describe("Constructor()", () => {
+		it("creates a handler for MESSAGE_Create", () => {
```

Comment:
`MESSAGE_CREATE`

Reply:
lowercase for consistency. see LogDeleteHandlerTest.ts

Reply:
Ah, that should be in uppercase too. Must have missed it.